src/index/validator/route.ts

```diff
@@ -24,21 +24,85 @@
  *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import { Type } from 'avsc';
-import * as HTTP_STATUS_CODE from 'http-status-codes';
+import { RequestHandler } from 'express-serve-static-core';
+import { BAD_REQUEST, OK } from 'http-status-codes';
 import _ from 'lodash';
 
-import { Options } from '../..';
+import { AvroSchema, Options, Request, Response, ResponseWriterFunction } from '../..';
+import { Server } from '../server';
 import * as RouteMethod from '../server/routeMethod';
 import { SchemaValidator } from './shared/schema';
 
+export interface RouteConfiguration {
+	apiEndpoint: string;
+	endpoint: string;
```

Comment:
I don't think endpoint is required in here any more just in the raw options.

Reply:
Removed